vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -207,6 +209,13 @@ func verifyAddSubnetValidatorTx(
 		return err
 	}
 
+	if backend.Config.UpgradeConfig.IsEtnaActivated(currentTimestamp) {
+		_, _, err := chainState.GetSubnetManager(tx.Subnet)
```

Comment:
`database.ErrNotFound` means that there is definitively no subnet manager. `nil` or any other error means there is a subnet manager. The details of the error aren't required since the only other possible error is a codec unmarshal error which could only happen if the db is corrupted.